jenkins-integration/jenkins-jobs/scenarios/puppetserver-infinite/JobDSL.groovy

```diff
@@ -0,0 +1,5 @@
+job.parameters {
+    stringParam('NUMBER_OF_HOURS', '2', 'Length of time to run the gatling simulation.')
+    choiceParam('CATALOG_SIZE', ['MEDIUM', 'EMPTY'], 'Catalog to use in simulation.')
+    choiceParam('NODE_COUNT', ['600', '100', '200', '300', '900', '1200', '1500', '1800'], 'Number of nodes to include in simulation.')
```

Comment:
It's probably useful to include in the description that these numbers are selected such that gatling will distribute them evenly

Reply:
Yea. That's a good idea. I also need to PR updating the scenarios to even that out.

Reply:
Does "choiceParam" show up as a drop-down or something, or do you have to know what your options are?

Reply:
It does! If you go to http://puppetserver-perf-driver55.delivery.puppetlabs.net:8080/job/puppetserver-infinite/ and look at build with params you should see it

Reply:
That's cool, never seen that before. Is it possible in JJB, I wonder, and we just don't use it?

Reply:
Not sure. It wasn't listed as an option in the JobDSL docs, but it was in an example and the source code, so it's weird all around.